apps/site/assets/css/_footer.scss

```diff
@@ -9,47 +9,128 @@
   background-color: $footer-color;
   color: $white;
   min-width: 20rem;
-  padding-bottom: $base-spacing * 2;
-  padding-top: $base-spacing * 2;
+  padding-bottom: 24px;
+  padding-top: 24px;
```

Comment:
This padding-top should increase with larger breakpoints - to 32px (sm, md) then 40px (lg, xl).